subsys/bluetooth/controller/hci/hci.c

```diff
@@ -2576,7 +2586,7 @@ static void le_advertising_report(struct pdu_data *pdu_data, u8_t *b,
 
 		adv_info->addr.type = adv->tx_addr;
 		memcpy(&adv_info->addr.a.val[0], &adv->adv_ind.addr[0],
-		       sizeof(bt_addr_t));
+				sizeof(bt_addr_t));
```

Comment:
Fix indentation.

Reply:
Thanks for pointing this out, should now be fixed